Blog/templates/Blog/content.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+{% extends 'Blog/base.html' %}
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>{% block title %}Actual content {% endblock %}</title>
+</head>
+{% block pagecontent %}
+<body>
+
+{%if blog%}
+{% for blog in blog %}
```

Comment:
I commented on view function to fetch only a single item, not an iterator full of them.

Use blocks properly. I commented elsewhere.